tests/gtest/avifgainmaptest.cc

```diff
@@ -776,7 +776,7 @@ TEST(GainMapTest, NoGainMap) {
   EXPECT_EQ(decoded->gainMap, nullptr);
 }
 
-TEST(GainMapTest, DecodeGainMapGrid) {
+TEST(GainMapTest, DISABLED_DecodeGainMapGrid) {
```

Comment:
Please add (TODO?) comments to explain why these tests are disabled.

Reply:
Added a TODO for the issues.